src/amber/cli/templates/scaffold/controller/crecto/spec/controllers/{{name}}_controller_spec.cr.ecr

```diff
@@ -1,72 +1,110 @@
-<% field_name = @fields.first.name -%>
+<% params_name =  "#{@name.downcase}_params" -%>
+<% hash_name   =  "#{@name.downcase}_hash" -%>
+<% model_name  =  "create_#{@name.downcase}" -%>
```

Comment:
It shouldn't be necessary to modify the name, but if we do, use `@name.underscore` to be safe.